src/Symfony/Component/Workflow/Registry.php

```diff
@@ -61,7 +70,7 @@ public function get($subject, $workflowName = null)
         return $matched;
     }
 
-    private function supports(Workflow $workflow, SupportStrategyInterface $supportStrategy, $subject, $workflowName): bool
+    private function supports(WorkflowInterface $workflow, WorkflowSupportStrategyInterface $supportStrategy, $subject, $workflowName): bool
```

Comment:
calling `get()` after calling `add()` with a SupportStrategyInterface instance  would break here because of the typehint, it should be removed for now